### PR TITLE
[261457] Upgrade syn

### PR DESCRIPTION
**Pull request by aYukiSekiguchi**
https://acsmine.tok.access-company.com/redmine/issues/261457

### CODE REVIEW

File: core/core.ex [hunk 1]
```diff
@@ -34,7 +34,7 @@ defmodule AntikytheraCore do
       calculate_connection_trial_count_from_health_check_grace_period()
       |> establish_connections_to_other_nodes()
 
-      :syn.init()
+      :syn.add_node_to_scopes([:antikythera])
```
Comment (Double-oxygeN) [inline core/core.ex:37]: The comment below relates to the change.

```diff
diff --git a/core/core.ex b/core/core.ex
index a46d0ed..e28ad7a 100644
--- a/core/core.ex
+++ b/core/core.ex
@@ -29,7 +29,7 @@ defmodule AntikytheraCore do
     add_gears_dir_to_erl_libs()
     AntikytheraCore.FileSetup.setup_files_and_ets_tables()
     AntikytheraCore.Config.Core.load()
-    # Just to suppress log messages by :syn.init()
+    # Just to suppress log messages by `:syn.add_node_to_scopes/1`.
     if not Antikythera.Env.no_listen?() do
       calculate_connection_trial_count_from_health_check_grace_period()
       |> establish_connections_to_other_nodes()
```

**Review by Double-oxygeN — APPROVED** [review]
LGTM

File: core/core.ex (hunk 1 above)
Comment (aYukiSekiguchi) [reply core/core.ex:37]: Thank you. Fixed at https://github.com/access-company/antikythera/pull/225/commits/de0d2d53860c91698e474fc32da5347213875cea

**Review by irisTa56 — APPROVED** [review]
LGTM!

**Pull request merged**